lessons/specifics/eex.md

```diff
@@ -2,7 +2,7 @@
 layout: page
 title: Embedded Elixir (EEx)
 category: specifics
-order: 4
+order: 5
```

Comment:
@doomspork I think it makes more sense to order it from:

3 ETS
4 EEx
5 Mnesia

to

3 ETS
4 Mnesia
5 EEx

Where Mnesia comes straight after ETS.